pycbc/events/stat.py

```diff
@@ -1741,6 +1755,39 @@ def logsignalrate(self, stats, shift, to_shift):
         logr_s += numpy.log((self.curr_mchirp / 20.) ** (11. / 3.))
         return logr_s
 
+    def single(self, trigs):
+        """
+        Calculate the necessary single detector information
+
+        In this case the ranking rescaled (see the lognoiserate method here)
+        with the phase, end time, sigma, SNR, template_id and the
+        benchmark_logvol values added in. This also stored the current chirp
```

Comment:
This part of the comment does not make sense to me. We are talking about a single-detector ranking stat, so why are the phase and end time mentioned?

Reply:
I see there has been some code movement between `single()` and `rank_stat_coinc()`, perhaps that explains the confusion?

Reply:
This method does not just calculate the single detector ranking stat. It is also responsible for making sure that triggers have all the information attached to them that is needed for the coinc stat calculation, including phase and end time. In any case, Arthur has not actually edited the comment, the git diff is being thrown off by him reordering the methods.